src/pages/Home/Home.jsx

```diff
@@ -1,8 +1,8 @@
 import Button from 'components/common/Button/Button';
 import useTranslation from 'hooks/useTranslation';
 import { useLogoutMutation } from 'services/auth/auth';
-
-import logo from 'assets/logo.svg';
+import smiles from '../../assets/smiles.svg';
+import Map from '../../components/common/Map/Map';
```

Comment:
don't forge to import components using the alias from `babel.config`
```suggestion
import Map from 'components/common/Map/Map';
```

Reply:
same with assets